recipe/build.sh

```diff
@@ -22,7 +22,7 @@ cmake ${CMAKE_ARGS} -D CMAKE_INSTALL_PREFIX=$PREFIX \
       -D PCRE_ROOT=$PREFIX \
       -D ZLIB_ROOT=$PREFIX \
       -D CURL_ROOT=$PREFIX \
-      -D LAPACK_LIBRARIES=$PREFIX/lib/liblapack${SHLIB_EXT} \
+      -D BLA_VENDOR=Generic \
```

Comment:
@seisman, can you check if this change to use `-D BLA_VENDOR=Generic` makes sense. I'm not familiar with BLAS/LAPACK and all that, but found https://github.com/conda-forge/sundials-feedstock/issues/118#issuecomment-2753343197 and this seems to fix the `libmkl_intel_lp64.so.2: cannot open shared object file` error on linux-64?